bodhi/server/push.py

```diff
@@ -47,6 +47,9 @@
 @click.option('--resume', help='Resume one or more previously failed pushes',
               is_flag=True, default=False)
 def push(username, cert_prefix, config, **kwargs):
+    if not username:
+        raise Exception("Please provide --username")
```

Comment:
I think there's a way to let click do this for you by declaring the option as required. I'll see if I can find it in their docs.